cli.js

```diff
@@ -6,11 +6,12 @@ import optimize from "./optimize.js";
 import zeebe from "./zeebe.js";
```

Comment:
❓ Just out of curiosity: do we need a `zeebe.js`? We don't use the Zeebe REST API, do we? Could the functions in there simply go to the `camunda-8.js`?

Reply:
This is actually for the Zeebe REST tutorial we created when it was still active :) https://docs.camunda.io/docs/8.5/apis-tools/zeebe-api-rest/zeebe-api-tutorial/